app.js

```diff
@@ -3,7 +3,9 @@ const Discord = require('discord.js');
 const cron = require('node-cron')
 
 const twitterService = new TwitterService('tholl_22');
-const client = new Discord.Client();
+const client = new Discord.Client({
+  intents: ['GUILDS', 'GUILD_MESSAGES', 'GUILD_PRESENCES']
+});
```

Comment:
I think I'm on an older version of the discordjs package, trying removing this and see if it still works. Idk if the version I'm on has introduced intents

Reply:
it works without them